core/bones/key.py

```diff
@@ -60,7 +64,7 @@ def singleValueFromClient(self, value, skel, name, origData):
 
     def unserialize(self, skel: 'viur.core.skeleton.SkeletonValues', name: str) -> bool:
         """
-            Inverse of serialize. Evaluates whats
+            Inverse of serialize. Evaluates what's
             read from the datastore and populates
             this bone accordingly.
             :param name: The property-name this bone has in its Skeleton (not the description!)
```

Comment:
I would like to remove this docstring entirely, as it is a copy of the docstring in `BaseBone.unserialize()`.
I would like to keep maintenance overhead as low as possible.